arcgis-ios-sdk-samples/Augmented reality/Display scenes in tabletop AR/README.md

```diff
@@ -2,7 +2,7 @@
 
 Use augmented reality (AR) to pin a scene to a table or desk for easy exploration.
 
-![Scene content shown sitting on a surface, as if it were a 3D printed model](image1.png)
+![Scene content shown sitting on a surface, as if it were a 3D printed model](display-scene-tabletop.jpeg)
```

Comment:
The name of the image file is `display-scenes-tabletop.png`. Could you please change the name here and in `metadata.json` to match? Thanks!

Reply:
Oops, brain fart. All fixed now! @philium